cabal.project

```diff
@@ -34,6 +34,8 @@ optional-packages: vendored/*/*.cabal
 
 tests: true
 
+flags: +pedantic
```

Comment:
I'm not sure we should do this here, rather only do it in CI. It's *really* annoying to develop with `-Werror` on!

Reply:
hmm i prefer doing it right step by step instead a final cleaning out, but fair enough

Reply:
Running with `-Werror` means that if you have a binding that's temporarily unused then it's an error and nothing else will compile. It's really pretty bad IMO.